TypeCobol.LanguageServer/Completion Factory/CompletionFactoryHelpers.cs

```diff
@@ -150,24 +156,24 @@ public static IEnumerable<CompletionItem> CreateCompletionItemsForProcedures(IEn
                     new CompletionItem(string.Format("{0} {1} {2} {3}", procDisplayName, inputParams, inoutParams, outputParams));
                 completionItem.insertText = procIsPublic
                     ? inputParams != null
-                            ? string.Format("{0}::{1} INPUT", proc.VisualQualifiedName.Tail, proc.VisualQualifiedName.Head)
+                            ? string.Format("{0}::{1} {2}", proc.VisualQualifiedName.Tail, proc.VisualQualifiedName.Head, paramWithCase["INPUT"])
```

Comment:
Can you use the constant ParamInput here ?

Reply:
fixed - it's was an error
https://github.com/TypeCobolTeam/TypeCobol/tree/6d2502c2ac02e718a765ac3e549933b59b6ebb69